tasks/actorstate/actorstate_test.go

```diff
@@ -319,3 +322,19 @@ func (m *MockAPI) newEmptyRewardStateV0(currRealizedPower abi.StoragePower) (*sa
 func (m *MockAPI) newEmptyRewardStateV2(currRealizedPower abi.StoragePower) (*sa2reward.State, error) {
 	return sa2reward.ConstructState(currRealizedPower), nil
 }
+
+func (m *MockAPI) newEmptyInitStateV0() (*sa0init.State, error) {
```

Comment:
should MockAPI move to it's own file?

Reply:
Definitely should. Could also accept a `*testing.T` and stop returning errors -- going to make these fixups my next PR.